lib/resources/r4/appointment.yaml

```diff
@@ -25,6 +25,36 @@ fields:
     }
   note: When creating a new appointment, the status must be `booked` or `proposed`.
 
+- name: cancelationReason
+  required: 'No'
+  type: CodeableConcept
+  description: The coded reason for the appointment being cancelled.
+  action: read
+  example: |
+    {
+      'cancelationReason': {
+        'coding': [
+          {
+            'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14229',
+            'code': '2191414701',
+            'display': 'Scheduled by mistake',
+            'userSelected': true
+          }
+        ],
+        'text': 'Scheduled by mistake'
+      }
+    }
+  url: https://hl7.org/fhir/R4/appointment-definitions.html#Appointment.cancelationReason
+  binding:
+    description:  The coded reason for the appointment being cancelled.
+    terminology:
+      - display: Appointment cancellation reason
+        system: http://hl7.org/fhir/ValueSet/appointment-cancellation-reason
+        info_link: https://www.hl7.org/fhir/r4/valueset-appointment-cancellation-reason.html
```

Comment:
Per my comment here: https://github.com/cerner/fhir.cerner.com/pull/584#discussion_r596925269

This should be: http://terminology.hl7.org/CodeSystem/appointment-cancellation-reason

Reply:
I had asked and verified the same with Alex and modified it the way he wanted.

Reply:
Talked to Alex again, and he is good with the one you suggested. Changed it.
https://github.com/cerner/fhir.cerner.com/commit/e800dbb7097efa29d479f05847ed49fd552dc444#diff-256db1c27cbe3136db6001a65a04bd3cffaf5beb61789a01f2c6da8b2676bc73R53